ts-apps/decision-logic-visualizer/src/lib/layout-ir/ladder-graph/edge-attributes.ts

```diff
@@ -0,0 +1,195 @@
+import type { Eq } from '$lib/utils.js'
+import 'core-js/actual/set'
+
+/********************************
+      Attributes
+*********************************/
+// This isn't in principle limited to edges;
+// it's just that we will prob use LirNode methods for node data
+
```

Comment:
The main thing that's new here is the EdgeStylesContainer stuff. The diff isn't the clearest because i moved the file